dev/com.ibm.ws.install.featureUtility/resources/com/ibm/ws/install/featureUtility/internal/resources/FeatureUtilityToolOptions.nlsprops

```diff
@@ -0,0 +1,89 @@
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+# NLS_MESSAGEFORMAT_VAR
+#--------- ACTIONS ----------------------------------------------------#
+# List of descriptive text for actions. Use "\n\" for line continuations.
+# The code provides no whitespace: begin keys with "\ \ \ \ " for leading
+# whitespace, and begin descriptions with \t
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
+usage=\
+Usage: {0}
+
+global.description=Description:
+global.actions=Actions:
+global.options=Options:
+global.options.lower=options
+global.usage=Usage:
+global.options.statement=\
+\tUse help [actionName] for detailed option information of each action.
+task.unknown=Unknown action: {0}
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
+help.desc=\
+\tPrint help information for the specified action.
+
+#--------- INSTALLFEATURE ACTION ---------------------------------------------#
+installFeature.desc=\
+\tInstall one or a list of space separated features.
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
+installFeature.option-key.name...=\
+\ \ \ \ name ...
+installFeature.option-desc.name...=\
+\tOne or more features separated by a space. For features, specify     \n\
+\tthe feature shortname or the maven coordinate. If specifying a       \n\
+\tmaven coordinate, it must contain at least the groupId and           \n\
+\tthe artifactId (e.g io.openliberty.features:mpHealth-2.0). The       \n\
+\tversion will default to the current runtime version and the          \n\
+\tpackaging type will be default to "esa" if they are not specified.
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
+installFeature.option-key.--to=\
+\ \ \ \ --to=[usr|extension]
+installFeature.option-desc.--to=\
+\tSpecify where to install the feature. The feature can be installed to\n\
+\tany configured product extension location, or as a user feature. If  \n\
+\tthis option is not specified the feature will be installed as a user \n\
+\tfeature.
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
+installFeature.option-key.--from=\
+\ \ \ \ --from=filePath
+installFeature.option-desc.--from=\
+\tUse this option to specify the file path of a Maven artifact         \n\
+\trepository as a source for featureUtility to install assets.
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
+installFeature.option-key.--verbose=\
+\ \ \ \ --verbose
+installFeature.option-desc.--verbose=\
+\tDisplay additional debug information while the action runs.
+#------------------------------\n at 72 chars -- leading tab-----------\n\#
+installFeature.option-key.--noCache=\
+\ \ \ \ --noCache
+installFeature.option-desc.--noCache=\
+\tFeatures installed from remote repository (e.g. Maven Central) will  \n\
+\tnot be cached locally.
```

Comment:
Update this section to read:
"Features installed from a remote repository (for example, Maven Central) are not cached locally."